tests/phpunit/api/v3/ACLPermissionTest.php

```diff
@@ -664,4 +664,20 @@ protected function getActivityContacts($activity) {
     return $contacts;
   }
 
+  /**
+   * Test that the 'everyone' group can be given access to a contact.
+   */
+  public function testGetACLEveryonePermittedEntity() {
+    $this->setupScenarioCoreACLEveryonePermittedToGroup();
+    $this->callAPISuccessGetCount('Contact', [
+      'id' => $this->scenarioIDs['Contact']['permitted_contact'],
+      'check_permissions' => 1,
+    ], 1);
+    $this->setupScenarioCoreACLEveryonePermittedToGroup();
```

Comment:
@eileenmcnaughton why need the 2nd call here?

Reply:
gone